stdlib/2/typing.pyi

```diff
@@ -385,7 +385,8 @@ class Pattern(Generic[AnyStr]):
 
 # Functions
 
-def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+def get_type_hints(obj: Callable, globalns: Optional[dict[AnyStr, Any]],
```

Comment:
Why should be `{u'key1': 'value1', 'key2': 'value2'}` incorrect for users?
btw python3 with {b'key': 'value'} works.